browser/layout/pa11y-config.js

```diff
@@ -6,19 +6,19 @@ module.exports = {
 	pa11yData: [
 		{
 			rootElement: 'html',
-			hideElements: '.n-skip-link',
+			hideElements: '.n-skip-link, iframe[src*=doubleclick]',
 			page: {
 				headers: {
-					'FT-Flags': 'ads:off,javascript:off'
+					'FT-Flags': 'ads:off,javascript:off,enableGTM:off'
```

Comment:
Is this belt and braces? Am I right to think there should be no ad iframe if the ads and GTM flags are off?

Reply:
It's not. For now, gpt.js is still loaded by o-ads and ads still load when the `enableGTM` flag is off. We might change this in the future, but not yet.